pkg/operator/ceph/csi/version_test.go

```diff
@@ -96,6 +100,23 @@ func TestSupported(t *testing.T) {
 	assert.Equal(t, false, ret)
 }
 
+func TestSupportOMAPController(t *testing.T) {
+	AllowUnsupported = false
+	ret := testMinVersion.SupportOMAPController()
+	assert.False(t, ret)
+
+	ret = testReleaseV300.SupportOMAPController()
+	assert.False(t, ret)
+
+	ret = testReleaseV320.SupportOMAPController()
+	assert.True(t, ret)
+
+	ret = testReleaseV321.SupportOMAPController()
+	assert.True(t, ret)
+
+	ret = testReleaseV330.SupportOMAPController()
+	assert.True(t, ret)
```

Comment:
Add a test for what happens when `AllowUnsupported = true`?

Reply:
done